tests/pact/caps.repl

```diff
@@ -41,6 +45,14 @@
   (defun enforce-msg-keyset (key:object{guardschema})
     (enforce-keyset (read-keyset (at 'key key))))
 
+  ; a user guard which reads from the db, which we want to disallow.
+  (defun db-user-guard ()
+    (insert ints 'x {'i: 0})
+    (create-user-guard {'k: 'x} "read-zero-from-db"))
+  (defun read-zero-from-db (o:object{ints-key})
```

Comment:
Should we also have a test of this being used successfully?

Reply:
You can find a test that succeeds here: https://github.com/kadena-io/pact/blob/0c64619af5e7333fd3d8fcccc524c0cf95f95f94/tests/pact/caps.repl#L154-L156